src/wp-includes/functions.php

```diff
@@ -3835,7 +3835,7 @@ function _deprecated_function( $function, $version, $replacement = null ) {
 	 *
 	 * @param string $function    The function that was called.
 	 * @param string $replacement The function that should have been called.
-	 * @param string $version     The version of WordPress that deprecated the function.
+	 * @param string $version     The version of ClassicPress that deprecated the function.
```

Comment:
Let's say "The version of ClassicPress or WordPress" for now.  We'll prefix these version numbers with `WP-` if they are WordPress versions.  See #27.